test/conftest.py

```
@@ -137,3 +139,128 @@ def is_rtsp_stream_ready(rtsp_address):
         except subprocess.TimeoutExpired:
             process.kill()
             process.wait()
+
+
+def wait_for_port(host: str, port: int, timeout: float = 10.0) -> None:
+    start = time.time()
+    while time.time() - start < timeout:
+        try:
+            with socket.create_connection((host, port), timeout=0.5):
+                return
+        except Exception:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)